### PR TITLE
Clean up the code

### PR DESCRIPTION
**Pull request by poncovka**

### CODE REVIEW

**Review by jkonecny12 — COMMENTED** [review]
Looks good although Martin should be the main reviewer here.

File: initial_setup/common.py [hunk 1]
```diff
@@ -94,46 +97,6 @@ def list_usable_consoles_for_tui():
...
-
-       :return: dictionary mapping of line
-       :rtype: dictionary
-    """
-    osrel = {}
-
-    for line in osreleaselines:
-        line = line.strip()
-        try:
-            key, sep, value = line.parition('=')
```
Comment (VladimirSlavik) [inline initial_setup/common.py:108]: Interesting. This could never have worked!

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by poncovka** [comment]
Tested with kickstart tests.

**Review by jkonecny12 — APPROVED** [review]
LGTM! I hope I'll enable pylint here soonish...

**Review by M4rtinK — COMMENTED** [review]
Looks good to me as well, thanks a lot! :)

**Pull request merged**